Build Error

cc -Werror -Wall -Wundef -Wmissing-prototypes -Wmissing-declarations -Wstrict-prototypes -Wold-style-definition -g -fstack-protector -I ccan -I secp256k1/include/ -I .    -c -o daemon/peer.o daemon/peer.c
daemon/peer.c: In function ‘their_htlc_added’:
daemon/peer.c:943:10: error: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 2 has type ‘u64 {aka long long unsigned int}’ [-Werror=format=]
   printf("FORWARDING %lu\n", step->hoppayload->amount);
          ^
cc1: all warnings being treated as errors
<builtin>: recipe for target 'daemon/peer.o' failed
make: *** [daemon/peer.o] Error 1

--- a/daemon/peer.c
+++ b/daemon/peer.c
@@ -940,7 +940,6 @@ static void their_htlc_added(struct peer *peer, struct htlc *htlc,
 		goto free_packet;
 
 	case ONION_FORWARD:
-		printf("FORWARDING %lu\n", step->hoppayload->amount);
 		route_htlc_onwards(peer, htlc, step->hoppayload->amount, step->next->nexthop,
 				   serialize_onionpacket(step, peer->dstate->secpctx, step->next), only_dest);
 		goto free_packet;

